plugin/core/sessions.py

```diff
@@ -542,6 +560,9 @@ def set_semantic_tokens_pending_refresh(self, needs_refresh: bool = True) -> Non
     def get_semantic_tokens(self) -> List[Any]:
         ...
 
+    def do_inlay_hints_async(self, view: sublime.View) -> None:
```

Comment:
Existing convention in this interface has those handlers starting with `on_*`.

Reply:
the `on_*` convention doesn't sound right,
we use `on_*` convention for `on_diagnostics_async`, but that is for diagnostics that are published from the server and not requested  by the client.

for me the `do_inlay_hints_async` is more appropriate because the client request things from the server.

if you still want me to rename this, just leave a comment. :)

Reply:
btw, the do_* convention was also used for the do_semantic_tokens_async function